src/NavigationDots/NavigationDots.tsx

```diff
@@ -52,7 +52,7 @@ export const NavigationDots = React.forwardRef<
 
   const width = visibleDots.reduce(
     (acc, el, index) =>
-      acc + (el.active ? 16 : el.small ? 6 : 8) + (index > 0 ? 4 : 0),
+      acc + (el.active ? 16 : el.small ? 8 : 10) + (index > 0 ? 4 : 0),
```

Comment:
These numbers looks wrong. It should be:
```suggestion
      acc + (el.active ? 16 : el.small ? 4 : 8) + 6,
```
`6` is for 2 times the margin. The check for the index can be removed, by starting the count at `-6` instead of `0`.

Reply:
Besides, I don't really see the point of setting the width of the container... Do you maybe have an idea?

Reply:
@alexisfontaine No you're right, I didn't realise but it seems quite unnecessary...